src/riak_kv_wm_json.erl

```diff
@@ -18,7 +18,17 @@
 %% %CopyrightEnd%
 %% % @format
 %%
--module(riak_kv_wm_otpjson).
+-module(json).
+-moduledoc """
+A library for encoding and decoding JSON.
+
+This module implements EEP68.
+
+Both encoder and decoder fully conform to
+[RFC 8259](https://tools.ietf.org/html/rfc8259) and
+[ECMA 404](http://www.ecma-international.org/publications/standards/Ecma-404.htm)
+standards. The decoder is tested using [JSONTestSuite](https://github.com/nst/JSONTestSuite).
+""".
```

Comment:
The triple quotes """ have a new meaning in OTP27 and therefore you get a warning when compiling this code with OTP26... whereas it actually is what you want it won't compile :-(.
 
 Since you want to drop the code in OTP27, you probably want to have it still compile without warnings in OTP26.
 
 ```
%% @doc
%% A library for encoding and decoding JSON.
%%
%% This module implements EEP68.
%%
%% Both encoder and decoder fully conform to
%% [RFC 8259](https://tools.ietf.org/html/rfc8259) and
%% [ECMA 404](http://www.ecma-international.org/publications/standards/Ecma-404.htm)
%% standards. The decoder is tested using [JSONTestSuite](https://github.com/nst/JSONTestSuite).
```